### ISSUE TITLE
[Bug]: Application Menu not showing on Windows OS

### ISSUE THREAD

**Issue by drhoussem** [NativePHP/laravel#87]
### What happened?

The menu is registred in Service Provider but it's not showing

### How to reproduce the bug

Register a menu in NativeAppServiceProvider.php
```
Menu::new()
            ->submenu(
                'About',
                Menu::new()
                    ->link('https://test.co', 'Test')
            )
            ->fileMenu()
            ->register();
```

Run `php artisan native:serve`

The app will show on screen but application menu is not showing

### Package Version

0.2.0

### PHP Version

8.1

### Laravel Version

10.x

### Node Version

18.16

### Which operating systems does with happen with?

Windows

**Comment by wattsie** [comment]
Do you happen to have defined a Menubar?

In latest update, I find that if I have a "MenuBar::create()" defined, the menus do no appear, as soon as I comment out the menubar, the main app menu appears again...

**Comment by drhoussem** [comment]
This was not a bug, it was the default behavior because `autoHideMenuBar` was set to true in electron plugin.
I worked on a feature to make this optional locally and I was about to make a pull request till I saw that there was already a PR #84  that introduced this feature (merged in 0.3.0).
